gulp.nginclude.js

```diff
@@ -0,0 +1,86 @@
+/* jshint node:true */
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var cheerio = require('cheerio');
+var glob = require('glob');
+var gutil = require('gulp-util');
+var through = require('through2');
+//var extend = require('lodash.assign');
+var extend = require('util')._extend;
+var PluginError = gutil.PluginError;
+
+module.exports = function(options) {
+    options = extend({trim: true}, options);
+
+    function transform(file, encoding, next) {
```

Comment:
It's gulp plugin - gulp-nginclude. I took source code and modified it to work with themes.
Should I add comments and refactor this code or just provide a link to the plugin and describe my changes?